src/cftime/_cftime.pyx

```diff
@@ -1237,6 +1236,32 @@ The default format of the string produced by strftime is controlled by self.form
             format = self.format
         return _strftime(self, format)
 
+    @staticmethod
+    def strptime(datestring, format, calendar='standard', has_year_zero=None):
+        """
+        Return a datetime corresponding to date_string, parsed according to format, 
+        with a specified calendar and year zero convention. 
+        For a complete list of formatting directives, see section
+        'strftime() and strptime() Behavior' in the base Python documentation.
```

Comment:
It might be good to document the subset of directives that are guaranteed to be supported for all calendars through the fallback internal implementation of `strptime`.

Reply:
now reads
```python
        """
        Return a datetime corresponding to date_string, parsed according to format,
        with a specified calendar and year zero convention.
        The format directives 'y','Y','m','B','b','d','H','M','S' and 'f'
        are supported for all calendars and dates.  If the date is valid
        in the python 'proleptic_gregorian' calendar, then python's
        datetime.strptime is used. For a complete list of formatting directives
        supported in python's datetime.strptime, see section
        'strftime() and strptime() Behavior' in the base Python documentation.
        """
```

Reply:
Great, thanks.  Looks good to me.